ui/src/app/utils/runtime-utils.tsx

```diff
@@ -1277,11 +1279,16 @@ export enum PanelContent {
 
 // should we show the runtime in the UI?
 export const isVisible = (status: RuntimeStatus) =>
-  status && ![RuntimeStatus.Deleted, RuntimeStatus.Error].includes(status);
+  status &&
+  !(
+    [RuntimeStatus.DELETED, RuntimeStatus.ERROR] as Array<RuntimeStatus>
+  ).includes(status);
 
 // is the runtime in a state where the user can take action?
 export const isActionable = (status: RuntimeStatus) =>
-  [RuntimeStatus.Running, RuntimeStatus.Stopped].includes(status);
+  (
+    [RuntimeStatus.RUNNING, RuntimeStatus.STOPPED] as Array<RuntimeStatus>
+  ).includes(status);
 
 export const getCreator = (runtime: ListRuntimeResponse): string | undefined =>
-  runtime?.labels?.creator;
+  (runtime?.labels as Map<string, string>).get('creator');
```

Comment:
Threw an error that "labels" is type object and object does not have a creator property. [The description of ListRuntimeResponse](https://github.com/all-of-us/workbench/blob/2fbf63c2f3ff8e070fac0e096edce395b01ab303/api/src/main/resources/workbench-api.yaml#L6870) shows that is actually of type Map<string, string>. Casted this in order to avoid the error.